projects/plugins/boost/app/modules/optimizations/minify/class-minify-js.php

```diff
@@ -12,7 +12,10 @@ class Minify_JS implements Pluggable {
 	public function setup() {
 		require_once JETPACK_BOOST_DIR_PATH . '/app/lib/minify/functions-helpers.php';
 
-		jetpack_boost_minify_setup();
+		$setup_done = jetpack_boost_minify_setup();
```

Comment:
I'm not 100% sure I agree with `$setup_done` naming here, although I don't have better suggestions right now 🤔

Maybe $minify_available? $can_minify? $should_minify?

Reply:
Yeah, makes sense. I didn't think twice, just copied the variable name from the function. To be honest, I don't like the function name as well...

Anyway, I've updated this in f37adb48a1e3b5c7f4355693fbf5988f5f21db55.